README.md

```diff
@@ -68,18 +82,18 @@ From the working directory:
    your actual LocalStack key.
 
       ```bash
-      export LOCALSTACK_API_KEY=<your key>
+      export LOCALSTACK_AUTH_TOKEN=<your key>
```

Comment:
Maybe change <your key> to <your auth token> so it's not confused with the deprecated api keys

Reply:
Change to what?

Reply:
@cabeaulac bumping this up.

Reply:
I mean change 
```
export LOCALSTACK_AUTH_TOKEN=<your key>
```
to
```
export LOCALSTACK_AUTH_TOKEN=<your auth token>
```

sorry for the confusion